src/tests/ftest/pool/bad_evict.py

```diff
@@ -94,7 +94,7 @@ def test_evict(self):
             # setup the DAOS python API
             with open('../../../.build_vars.json') as build_file:
                 data = json.load(build_file)
-            context = DaosContext(data['PREFIX'] + '/lib/')
+            context = DaosContext(data['PREFIX'] + '/lib64/')
```

Comment:
Since this test inherits TestWithServers, this code is already executed in `TestWithServers.setup()` and the `DaosContext` object is assigned to `self.context`.  That means the pool creation can simple be `pool = DaosPool(self.context)` and all the code using `json` (including the import) and assigning `context` can be removed.

To make use of the existing `TestWithServers.tearDown()` functionality the `pool` variable should be `self.pool`.